src/components/button-toggle-group/__definition__.js

```diff
@@ -0,0 +1,30 @@
+import ButtonToggleGroup from './';
+import Definition from './../../../demo/utils/definition';
+
+const definition = new Definition('button-toggle-group', ButtonToggleGroup, {
+  description: 'Adds support for validation, warning and info messages on a group of Button Toggles.',
+  type: 'form',
+  propValues: {
+    children: `
+<ButtonToggle name={ 'grouped-button-toggle' } id={ 'foo' } value={ 'foo' }>Foo</ButtonToggle>
+<ButtonToggle name={ 'grouped-button-toggle' } id={ 'bar' } value={ 'bar' }>Bar</ButtonToggle>
+<ButtonToggle name={ 'grouped-button-toggle' } id={ 'baz' } value={ 'baz' }>Baz</ButtonToggle>
```

Comment:
this is rendering with some additional line breaks and missing indentation if you check the demo code that is rendered

Reply:
I know. Any ideas on how to stop it - there's no heredoc support in JS is there?

Reply:
You just need to clear/add whitespace and line breaks where it needs it.